ebos/eclgenericthresholdpressure.cc

```diff
@@ -77,14 +79,9 @@ thresholdPressure(int elem1Idx, int elem2Idx) const
 
     // threshold pressure accross faults
     if (!thpresftValues_.empty()) {
-        int cartElem1Idx = cartMapper_.cartesianIndex(elem1Idx);
-        int cartElem2Idx = cartMapper_.cartesianIndex(elem2Idx);
-
-        assert(0 <= cartElem1Idx && static_cast<int>(cartElemFaultIdx_.size()) > cartElem1Idx);
-        assert(0 <= cartElem2Idx && static_cast<int>(cartElemFaultIdx_.size()) > cartElem2Idx);
```

Comment:
This check is now done inside of LookUpCartesianData, right?

Reply:
Indeed, lines 323-324 (also lines 313-14) in LookUpData.hh